include/rocksdb/utilities/backupable_db.h

```diff
@@ -122,16 +135,23 @@ struct BackupableDBOptions {
   // Default: INT_MAX
   int max_valid_backups_to_open;
 
-  // If true, backup SST filenames consist of file_number, crc32c, db_session_id
-  // if false, backup SST filenames consist of file_number, crc32c, file_size
+  // Naming option for share_files_with_checksum table files.
+  // The default behavior for this option fixes the backup file name collision
```

Comment:
In this and the next paragraph, it feels like we're focusing on the fix/change. But for people reading the code (rather than a PR), it's probably not necessary or easy for them to know about a past change, especially as more time passes. Maybe we can describe it in a more time-independent way with statements like "`kChecksumAndFileSize` is susceptible to collision", "`kChecksumAndDbSessionId` is immune to collision", "Modifying this option cannot introduce a downgrade compatibility issue", etc.

Reply:
Good point :)